### ISSUE TITLE
Bug: font-size: clamp() reported as error incorrectly

### ISSUE THREAD

**Issue by mgoetz-nerdery** [eslint/css#159]
### Environment

ESLint version: 9.25.0
@eslint/css version: 0.8.1
Node version: 22.13.1
npm version: 10.9.2
Operating System: Mac OS 15

### Which language are you using?

stylesheet

### What did you do?

<details>
<summary>Configuration</summary>

```
import js from "@eslint/js";
import globals from "globals";
import reactHooks from "eslint-plugin-react-hooks";
import reactRefresh from "eslint-plugin-react-refresh";
import tseslint from "typescript-eslint";
import css from "@eslint/css";

export default tseslint.config(
  { ignores: ["dist"] },
  {
    extends: [js.configs.recommended, ...tseslint.configs.recommended],
    files: ["**/*.{ts,tsx}"],
    languageOptions: {
      ecmaVersion: 2020,
      globals: globals.browser,
    },
    plugins: {
      "react-hooks": reactHooks,
      "react-refresh": reactRefresh,
    },
    rules: {
      ...reactHooks.configs.recommended.rules,
      "react-refresh/only-export-components": [
        "warn",
        { allowConstantExport: true },
      ],
    },
  },
  {
    files: ["**/*.css"],
    language: "css/css",
    ...css.configs.recommended,
  },
);
```
</details>

```css
  .fluidHeading {
    font-size: clamp(2.1rem, calc(7.2vw - 0.2rem), 2.5rem);
  }
```

### What did you expect to happen?

I expected that this would lint with no errors.

### What actually happened?

  3:16  error  Invalid value 'clamp(2.1rem,calc(7.2vw - 0.2rem),2.5rem)' for property 'font-size'. Expected <absolute-size> | <relative-size> | <length-percentage [0,∞]> | math  css/no-invalid-properties

✖ 1 problem (1 error, 0 warning)

### Link to Minimal Reproducible Example

https://stackblitz.com/edit/vitejs-vite-du8wqxzy?file=src%2Fstyle.css

### Participation

- [ ] I am willing to submit a pull request for this issue.

**Comment by nzakas** [comment]
Thanks for the reproduction case, I can reproduce this. This is most likely related to the syntax definitions coming from CSSTree. I'll look into that.

**Comment by mgoetz-nerdery** [comment]
No rush!  I have a workaround using media queries for the time being but I'm definitely interested when this is resolved.

**Comment by nzakas** [comment]
Looks like `min()` and `max()` also don't work. It appears CSSTree only ever implemented `calc()`. Note for myself:
https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Values_and_Units/Using_CSS_math_functions

**Comment by mgoetz-nerdery** [comment]
If I have the project right, looks like there is a similar reported issue in CSSTree: https://github.com/csstree/csstree/issues/338
